charts/values.yaml

```diff
@@ -180,6 +192,12 @@ clustersynchroManager:
   tolerations:
     - key: node-role.kubernetes.io/master
       operator: Exists
+  ## @param featureGate to clustersynchro
+  featureGates:
+    - name: PruneManagedFields
+      enabled: true
+    - name: PruneLastAppliedConfiguration
+      enabled: true
```

Comment:
I think it might be better to use the map form.
eg.
```yaml
  featureGates:
    PruneManagedFields: true
    PruneLastAppliedConfiguration: true
```
This way we can use `--set clustersynchroManager.featurGates.PruneManagedFields: true` to set it when installing or upgrading